src/draw/sw/blend/lv_draw_sw_blend_to_argb8888.c

```diff
@@ -96,90 +98,97 @@ LV_ATTRIBUTE_FAST_MEM void lv_draw_sw_blend_color_to_argb8888(_lv_draw_sw_blend_
     /*Simple fill*/
     if(mask == NULL && opa >= LV_OPA_MAX) {
 #ifdef LV_DRAW_SW_COLOR_BLEND_TO_ARGB8888
-        LV_DRAW_SW_COLOR_BLEND_TO_ARGB8888(dsc);
-#else
-        uint32_t color32 = lv_color_to_u32(dsc->color);
-        uint32_t * dest_buf = dsc->dest_buf;
-        for(y = 0; y < h; y++) {
-            for(x = 0; x < w - 16; x += 16) {
-                dest_buf[x + 0] = color32;
-                dest_buf[x + 1] = color32;
-                dest_buf[x + 2] = color32;
-                dest_buf[x + 3] = color32;
-
-                dest_buf[x + 4] = color32;
-                dest_buf[x + 5] = color32;
-                dest_buf[x + 6] = color32;
-                dest_buf[x + 7] = color32;
-
-                dest_buf[x + 8] = color32;
-                dest_buf[x + 9] = color32;
-                dest_buf[x + 10] = color32;
-                dest_buf[x + 11] = color32;
-
-                dest_buf[x + 12] = color32;
-                dest_buf[x + 13] = color32;
-                dest_buf[x + 14] = color32;
-                dest_buf[x + 15] = color32;
-            }
-            for(; x < w; x ++) {
-                dest_buf[x] = color32;
-            }
+        if(!LV_DRAW_SW_COLOR_BLEND_TO_ARGB8888(dsc))
```

Comment:
It could be even cleaner if in case of `LV_DRAW_SW_ASM_NONE` is defined these
```c
#defeine LV_DRAW_SW_COLOR_BLEND_TO_ARGB8888(dsc) false;
```

This way no extra `#ifdef` guard is required here.

Reply:
Good idea. I will make these change later.

Reply:
Done